libs/common/src/helpers/string.ts

```diff
@@ -0,0 +1,5 @@
+export const isString = (value: unknown): value is string => typeof value === 'string' || value instanceof String;
```

Comment:
Docstrings please.

Reply:
Should all the `value: any` here?

Reply:
Should we keep the signature to `: string`?

Reply:
> Should all the value: any here?

Using type of `unknown` gives us better type-safety. See my previous comment in the adjacet `string.test.ts` file.

> Should we keep the signature to : string?

The basic return type would be `: boolean`, but by using a [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) of `: value is string` we make the function much more powerful. This tells the TypeScript compiler that if the function returns true then the type of `value` is `string`, so you don't then need to cast it.

e.g. if we just returned `boolean` we would need to add a cast after returning `true` from the `isString` function, even though we now know the `value` is a `string`
```
if (isString(value)) {
  functionThatRequiresAString(value as string);
}
```
but with the `value is string` type predicate the compiler now knows `value` has type of `string` within the `if` block, so can just do
```
if (isString(value)) {
  functionThatRequiresAString(value);
}